lib/stat_tracker.rb

```diff
@@ -37,9 +38,48 @@ def lowest_total_score
       lowest_score = total_score if total_score < lowest_score
       lowest_score.round(1)
     end
-
     lowest_score
   end
 
+  def count_of_games_by_season
+    games_by_season = Hash.new(0)
+
+    @games_data.each do |game|
+      season = game[:season]
+      games_by_season[season] += 1
+    end
+    games_by_season
+  end
 
+  def average_goals_per_game
+    total_goals = 0
+    total_games = @games_data.length  
+
+    @games_data.each do |game|
+      total_goals += game[:home_goals].to_i + game[:away_goals].to_i
+    end
+    average_goals = total_goals.to_f / total_games
+    average_goals.round(2)
+  end
+
+  def average_goals_per_season
+    goals_by_season = Hash.new(0)
```

Comment:
wonder if we can store games by season somewhere because it looks like it's used on other methods